docs/index.md

```diff
@@ -15,8 +15,7 @@ hide:
 
 <div class="bitovi-row">
     <div class="bitovi-column">
-        <h1>Organize your Infrastructure-as-Code (IAC) 
-        <nl>Consistently Deploy.</h1>
+        <h1>BitOps packages, centralizes and organizes your Infrastructure-as-Code</h1>
```

Comment:
Does BitOps package your IaC?  I think of "packaging" as "building artifacts for deployment", and neither bitops nor the ops repo does that for your IaC.

What about just
> BitOps centralizes and organizes your Infrastructure-asCode

?

Reply:
How about replacing "packages" -> "deploys"?

```suggestion
        <h1>BitOps centralizes, organizes and deploys your Infrastructure-as-Code</h1>
```
In the past version there was mention of deployments.

Reply:
`deploys` is good.  I recommend adding the Oxford comma ;). @PhillypHenning